src/pages/home/report/ReportActionContextMenu.js

```diff
@@ -148,10 +153,26 @@ class ReportActionContextMenu extends React.Component {
     confirmDeleteAndHideModal() {
         deleteReportComment(this.props.reportID, this.props.reportAction);
         this.setState({isDeleteCommentConfirmModalVisible: false});
+        this.hidePopover();
     }
 
     hideDeleteConfirmModal() {
         this.setState({isDeleteCommentConfirmModalVisible: false});
+        this.hidePopover();
+    }
+
+    /**
+     * Hide the Menu
+     *
+     * @param {Boolean} delay whether the meanu should close after a delay
+     * @memberof ReportActionContextMenu
+     */
+    hidePopover(delay) {
```

Comment:
This should probably be called something like `hasDelay` or `shouldDelay` since it's a boolean.